docs/manual/inference.tex

```diff
@@ -505,18 +505,18 @@
 
 Each script repeatedly runs the checker with an \<-Ainfer=> command-line option to infer
 types for fields and method signatures.  The output of this step
-is a \<.jaif> (for \<infer-and-annotate.sh>) or stub (for \<wpi.sh>) file that records the inferred types.
+is a \<.jaif> (for \<infer-and-annotate.sh>) or \<.ajava> (for \<wpi.sh>) file that records the inferred types.
 Each script adds the inferred annotation to the next run, so that the checker takes them into
 account (and checks them). \<wpi.sh> does this using the \<-AmergeStubsWithSource> command-line
```

Comment:
This mentions `-AmergeStubsWithSource`. It's no longer necessary to specify this with ajava files. They behave similarly to the `-AmergeStubsWithSource` automatically.

Reply:
I've removed the reference to it. I also opened a PR to actually remove it from the wpi tool in DLJC (which we should have done during the PR I merged earlier today, but I didn't realize this!): https://github.com/kelloggm/do-like-javac/pull/49.